### ISSUE TITLE
Unbrick RIGOL Firmware updates

### ISSUE THREAD

**Issue by LinusCDE** [RPiList/specials#1806]
Please remove `beyondmeasure.rigoltech.com` from the adlists shown below.

![grafik](https://github.com/user-attachments/assets/fd7d691a-d1ea-4bfa-810f-2ebe39cca158)

I was wondering why my Oscilloscope couldn't update. Seems I got a severe case of over-blocking, and this is the main reason.

When I went to their site to download the firmware manually, the download failed, due to the domain being blocked.

![grafik](https://github.com/user-attachments/assets/7499a7d5-3500-4e4f-acf2-b507d79c907f)

I was gonna make a PR for it, but this Repo seems gigantic (4+ GB?!?).

**Comment by LizenzFass78851** [comment]
> Edit: Correction, this repo over 10 GB???

Yes. I noticed that too. Due to the huge accumulation of commits in which every commit saves every state of the files to date (which is normal with git). By that I mean that all lists in file form are probably significantly less than 10 GB in size and since the repo has existed since 2019 and since then over 11,000 commits have been committed through the list updates alone. As mentioned above, this means that every change is saved incrementally in the repo and therefore the repo uses a lot of memory. This is especially noticeable if you, as someone with 50 Mbits, have to clone the entire repo in order to be able to create pull requests, for example, which can take a while with the internet speed mentioned. @RPiList can take a look at the following https://github.com/orgs/community/discussions/73155#discussioncomment-7479336 which explains how to shink the git repo to save storage space.  I cannot say whether this will also be noticeable on the server side for anyone who re-clones the repo.

**Comment by RPiList** [comment]
> Please remove `beyondmeasure.rigoltech.com` from the adlists shown below.
> 
I'm afraid this is not a false positive. This Domain has been flagged by the Adguard Team as a combined disguised tracking domain. This is considered a form of malware.

When they delete their flag, it will automatically be deleted in our list.

Greetings

**Comment by RPiList** [comment]
In a different matter; it seems you have imported at least two of our lists multiple times.

Your picture shows our malware and spam.mails list two times.

The domain will be deleted from the spam.mail list because the problem is not spam. But, as stated above, it is malware.

**Comment by LinusCDE** [comment]
> I'm afraid this is not a false positive. This Domain has been flagged by the Adguard Team as a combined disguised tracking domain. This is considered a form of malware.
>
> When they delete their flag, it will automatically be deleted in our list.

Makes sense. This kinda sucks tbh. It's a really dirty technique. No idea how this should be handled. But since I always use uBO as well everywhere, I'd rather try to not overblock. Even if they misuse domains for both, randomly causing bugs in other devices in my network is not worth it.

> In a different matter; it seems you have imported at least two of our lists multiple times.
>
> Your picture shows our malware and spam.mails list two times.

I think, since the query took seemingly too long, I may have hit the search button twice, causing in duplicate results showing up.

Re-running and waiting only shows them once:

![grafik](https://github.com/user-attachments/assets/df1087c4-4ea2-4b9a-bbae-2ffdc8a6c46a)


> The domain will be deleted from the spam.mail list because the problem is not spam. But, as stated above, it is malware.

Nice. I'll also need to update my lists, maybe mark some categories as potential overblocking, so I can selectively turn them off/on for certain devices as needed.

I added the lists a couple of years ago and then barely touched it for a long time (running inside an docker container that auto-updates using watchtower).

![grafik](https://github.com/user-attachments/assets/6e2d1f71-5aae-444b-83f6-de1651fbe490)

Btw, great job! These lists are awesome nevertheless. Especially perfect for categorizing. I have 66 lists added and these ones are usually the notable ones and most useful for me due to the nice naming.

<details>

<summary>Pihole overview if interested</summary>

![grafik](https://github.com/user-attachments/assets/98ac44e7-5c3c-4c48-b034-226c3e993f1e)

Netflix app on the Samsung TV, even though just rarely used, it a pretty big offender. Next is the TV itself ofc.

I think some Huawei smartphones used to also be very high up on top blocked. But not using those anymore, so they don't show up here.

</details>
